src/modules/findProvinceFromCoordinate/index.ts

```diff
@@ -0,0 +1,74 @@
+import { GeoJSONData } from "./IRGeoJSON";
+
+interface Point {
+	longitude: number;
+	latitude: number;
+}
+
+interface GeoJSONFeature {
+	properties: Record<string, string | undefined>;
+	geometry: {
+		type: string;
+		coordinates: object; // This represents MultiPolygon coordinates
+	};
+}
+
+interface Province {
+	properties: {
+		'name:fa': string;
+		'name:en': string;
+	};
+	geometry: object; 
+}
```

Comment:
1. what is the point of showing users geometry of the given coordinates
- as the functions indicates, you will provide a set of coordinate and get a result respectively

it would be better to remove this property entirely and focus on the main purpose of this function which is finding a province 

something like:
```js
const { fa, en } = findProvinceFromCoordinate(...)
```

2. it should have a type anything other than raw object